src/test/java/io/reactivex/completable/CompletableTest.java

```diff
@@ -2796,11 +2796,43 @@ public void accept(Object e) { }
         });
     }
 
+    @Test(timeout = 5000)
+    public void asNormal() {
+        Flowable<Object> flow = normal.completable.as(new CompletableConverter<Flowable<Object>>() {
+            @Override
+            public Flowable<Object> apply(Completable c) {
+                return c.toFlowable();
+            }
+        });
+
+        flow.blockingForEach(new Consumer<Object>() {
```

Comment:
this does not assertion is this wanted?

Reply:
Deferring to @akarnokd on this as I was matching the analogous `to()` test

Reply:
Please change these into proper assertions such as `.test().assertResult()`.

Reply:
What would I assert for these since there's no value? Or just rewrite the rest in general?

Reply:
But there is the `onComplete` event.

Reply:
there is but `completable.normal` doesn't complete. I wasn't sure if you wanted me to change it to just use `Completable.complete()` or something similar

Reply:
`normal.completable` [does complete](https://github.com/hzsweers/RxJava/blob/5c3bdf37cd587df0ed4be769ede29e81f7b17d2f/src/test/java/io/reactivex/completable/CompletableTest.java#L110) otherwise `blockingForEach` would block indefinitely and the test would have failed a long a go. A `.test().assertResult(/* no values */)` should suffice.

Reply:
oops, I missed that line 🙄

Done in e0d793f